chromadb/api/collection_configuration.py

```diff
@@ -26,31 +25,24 @@ class HNSWConfiguration(TypedDict, total=False):
     resize_factor: float
 
 
-def default_hnsw_configuration() -> HNSWConfiguration:
-    return HNSWConfiguration(
-        space="l2",
-        ef_construction=100,
-        max_neighbors=16,
-        ef_search=100,
-        num_threads=cpu_count(),
-        batch_size=100,
-        sync_threshold=1000,
-        resize_factor=1.2,
-    )
+class SpannConfiguration(TypedDict, total=False):
+    search_nprobe: int
+    write_nprobe: int
+    space: Space
+    ef_construction: int
+    ef_search: int
+    max_neighbors: int
```

Comment:
honestly `m` was more readable since it is a standard hnsw param. So I would use a name such as `hnsw_m` for this and for the other two `hnsw_ef_construction` and `hnsw_ef_search`